app/stylesheets/layout.css

```diff
@@ -0,0 +1,68 @@
+.row,
+.column {
+	display: flex;
```

Comment:
please use https://prettier.io/ in your editor and/or `npm run lint` from command line

Reply:
@Aprillion I've just over a week ahead where I expect to work on Stampy quite a lot, and I've found these kinds of utility classes or whatever you want to call them a very good pattern for solving some common problems in CSS / HTML interactions so while I intend to make fairly extensive use of this stylesheet in coming days, even if I didn't I would at least advocate for moving in that direction anyway. Adam Wathan of Tailwind CSS does a better job explaining it than I do: https://adamwathan.me/css-utility-classes-and-separation-of-concerns/

All that being said, it's possible the actually better move is just using TailwindCSS instead.